setup.cfg

```diff
@@ -42,6 +42,8 @@ ignore=
   E221,
   # E252 - Spaces in parameters, a style deliberately avoided
   E252,
+  # E501 - Ignore max-line-length property - this is already defined by black
+  E501,
```

Comment:
This commit needs to be earlier for the previous commit to pass linting?

Reply:
Right, I had a bit of difficulty deciding if I should keep the first 2 commits together (they look related - the second one was the result of first?), or move this commit in between. Now, I realize that passing linting for each commit is more important and that's what my PR description indicates too. :+1: